src/__tests__/downshift.get-input-props.js

```diff
@@ -385,6 +385,19 @@ test('enter on an input with an open menu does nothing without a highlightedInde
   expect(childrenSpy).not.toHaveBeenCalled()
 })
 
+test.only('enter on an input with an open menu with 0 items calls submit', () => {
```

Comment:
Are you sure you want to use `.only` here?

Reply:
Nope, this definitely shouldn't be there. This was just here for ease of testing.

I can't seem to get this test to pass though. Would love to get some more eyes on this to figure out why, or if anyone has other suggestions on how to test this, I'd love to hear them too.